Makefile.core.mk

```diff
@@ -311,25 +339,25 @@ $(LOCALBIN):
 
 ## Tool Binaries
 OPERATOR_SDK ?= $(LOCALBIN)/operator-sdk
-KUSTOMIZE ?= $(LOCALBIN)/kustomize
+HELM ?= $(LOCALBIN)/helm
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 OPM ?= $(LOCALBIN)/opm
 
 ## Tool Versions
   OPERATOR_SDK_VERSION ?= v1.33.0
-  KUSTOMIZE_VERSION ?= v5.3.0
+  HELM_VERSION ?= v3.13.3
```

Comment:
do we need to install helm? It's already in the builder image.

Reply:
I guess we don't strictly need it here, but I don't use the builder image for local testing so it helps me

Reply:
ok, we can improve this in a follow up, e.g. by only installing it if it's not in $PATH. Not using the builder image is not an excuse to install every tool in the host. That's the whole purpose of the image :)

Reply:
We even have `oc` in the image nowadays :)